plugin/webpack.dev.js

```diff
@@ -0,0 +1,7 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: 'development',
+  devtool: 'inline-source-map',
```

Comment:
The critical line that allows the code to be debugged. See [the documentation](https://webpack.js.org/guides/development/).